packages/app/src/specs/SpecsList.vue

```diff
@@ -153,7 +153,9 @@
     v-model="isLoginOpen"
     :gql="props.gql"
     :utm-medium="loginUtmMedium"
-    @loggedin="refreshPage"
+    show-connect-button-after-login
```

Comment:
Going to test this logic a bit more myself, initially I figured just always showing it when you've logged in via the tooltip makes sense, but actually should probably still confirm no project ID, it's valid to log in from the tooltip into a connected project